matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/crypto/keysbackup/DefaultKeysBackupService.kt

```diff
@@ -1100,6 +1100,16 @@ internal class DefaultKeysBackupService @Inject constructor(
         return true
     }
 
+    override fun isValidRecoveryKeyForCurrentVersion(recoveryKey: String, callback: MatrixCallback<Boolean>) {
+        val keysBackupVersion = keysBackupVersion ?: return Unit.also { callback.onSuccess(false) }
```

Comment:
I'll be more confident to use a different name than the member class name for this val